pkg/porter/credentials.go

```diff
@@ -139,15 +140,10 @@ func (p *Porter) GenerateCredentials(opts CredentialOptions) error {
 
 // Validate validates the args provided Porter's credential show command
 func (o *CredentialShowOptions) Validate(args []string) error {
-	switch len(args) {
-	case 0:
-		return errors.Errorf("no credential name was specified")
-	case 1:
-		o.Name = strings.ToLower(args[0])
-	default:
-		return errors.Errorf("only one positional argument may be specified, the credential name, but multiple were received: %s", args)
+	if err := validateCredentialName(args); err != nil {
+		return err
 	}
-
+	o.Name = strings.ToLower(args[0])
```

Comment:
Credential set names are allowed by the spec to have upper case letters in the name. I'm not sure why we went with forcing `ToLower` here. 🤔 Seems to work but it's weird. I'm gonna chew on that for a while and figure out if it's a problem, a weird quirk that we can remove? @vdice if you have any context on this, I'm all ears 👂

Reply:
I just verified that we shouldn't be using `strings.ToLower`. It definitely blows up when using plugins (which care about case). Would you update this to take the arg as-is?